doc/cephfs/fs-volumes.rst

```diff
@@ -340,6 +340,12 @@ A clone can be in one of the following states:
 #. `in-progress` : Clone operation is in progress
 #. `complete`    : Clone operation has successfully finished
 #. `failed`      : Clone operation has failed
+#. `canceled`    : Clone operation is cancelled by user
```

Comment:
s/canceled/cancelled

Reply:
Unfortunately, the typo is in actual code itself. Leaving this for now.